22-12-2021 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionError al cargar la clave del certificadoCodigo error afip:*****Numero de Error: 1429Linea de codigo de error: cms = WSAA.SignTRA(tra, cRuta + this.pcrt, cRuta + this.pkey) && Cert. DemoProcedimiento: sw_wsaa_iniciarError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              
.Exepción: RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              
Traceback: Traceback (most recent call last):
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              

22-12-2021 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionError al cargar la clave del certificadoCodigo error afip:*****Numero de Error: 1429Linea de codigo de error: cms = WSAA.SignTRA(tra, cRuta + this.pcrt, cRuta + this.pkey) && Cert. DemoProcedimiento: sw_wsaa_iniciarError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              
.Exepción: RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              
Traceback: Traceback (most recent call last):
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsaa.pyc", line 258, in SignTRA
  File "wsaa.pyc", line 96, in sign_tra
RuntimeError: Archivos no encontrados: J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\gestion.key                                                                               , J:\XSOFTSQL\PROYECTOS\TRANSPORTE\AfipCert\ElCoyote.crt                                                                              

26-04-2022 17:35:06 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,1)Fecha Ulti: 26-04-2022ERROR AL SOLICITARCAEEl nro de comprobante autorizado 1 ya se encontraba informado.El ultimo autorizado por Afip es 1El Total consultado (0.00)no coincide con el del comprobante (1040.60)El CUIT consultado (20925289109)no coincide con el del comprobante (30716216183)
27-04-2022 14:42:41 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionSolicitando TATA Almacenado wsfeWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,56)Fecha Ulti: 26-04-2022Codigo error afip:*****Numero de Error: 1429Linea de codigo de error: cCAESolicitar = WSFE.CAESolicitar()Procedimiento: lote_procesoError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsfev1.pyc", line 288, in CAESolicitar
  File "pysimplesoap\client.pyc", line 148, in <lambda>
  File "pysimplesoap\client.pyc", line 352, in wsdl_call
  File "pysimplesoap\client.pyc", line 223, in call
SoapFault: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1208). ---> Input string was not in a correct format.
.Exepción: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1208). ---> Input string was not in a correct format.Traceback: Server was unable to read request. ---> There is an error in XML document (5, 1208). ---> Input string was not in a correct format.
27-04-2022 14:45:16 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,56)Fecha Ulti: 26-04-2022Codigo error afip:*****Numero de Error: 1429Linea de codigo de error: cCAESolicitar = WSFE.CAESolicitar()Procedimiento: lote_procesoError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsfev1.pyc", line 288, in CAESolicitar
  File "pysimplesoap\client.pyc", line 148, in <lambda>
  File "pysimplesoap\client.pyc", line 352, in wsdl_call
  File "pysimplesoap\client.pyc", line 223, in call
SoapFault: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1208). ---> Input string was not in a correct format.
.Exepción: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1208). ---> Input string was not in a correct format.Traceback: Server was unable to read request. ---> There is an error in XML document (5, 1208). ---> Input string was not in a correct format.
28-04-2022 13:48:31 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionSolicitando TATA Almacenado wsfeWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,0)Fecha Ulti: 28-04-2022Llamada: WSFE.CAESolicitarOrden:11000000076110058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
28-04-2022 14:09:35 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,0)Fecha Ulti: 28-04-2022Orden:11000000076110058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
28-04-2022 14:11:03 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,0)Fecha Ulti: 28-04-2022Orden:11000000076110058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
28-04-2022 14:13:28 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,0)Fecha Ulti: 28-04-2022Orden:11000000076110058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
28-04-2022 14:14:04 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,0)Fecha Ulti: 28-04-2022Orden:11000000076110058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
28-04-2022 14:16:01 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,0)Fecha Ulti: 28-04-2022Orden:11000000076110058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
28-04-2022 14:16:48 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,0)Fecha Ulti: 28-04-2022Orden:11000000076110058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
28-04-2022 14:18:09 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,0)Fecha Ulti: 28-04-2022Orden:11000000076110058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
28-04-2022 18:40:38 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,160)Fecha Ulti: 28-04-2022ERROR AL SOLICITARCAEEl nro de comprobante autorizado 160 ya se encontraba informado.El ultimo autorizado por Afip es 160El Total consultado (0.00)no coincide con el del comprobante (1839.20)
28-04-2022 18:44:04 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,160)Fecha Ulti: 28-04-2022ERROR AL SOLICITARCAEEl nro de comprobante autorizado 160 ya se encontraba informado.El ultimo autorizado por Afip es 160El Total consultado (0.00)no coincide con el del comprobante (1839.20)
28-04-2022 18:48:45 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,160)Fecha Ulti: 28-04-2022Codigo error afip:*****Numero de Error: 12Linea de codigo de error: =SaveSQL(WSFE.XmlResponse,"WSFE_CompConsultar"+cTipoCbte+"_"+cPunto_Vta+"_"+cCbte_Nro)Procedimiento: lote_procesoError VFP: No se encuentra la variable 'CTIPOCBTE'.Exepción: Traceback: 
28-04-2022 18:49:58 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,160)Fecha Ulti: 28-04-2022Codigo error afip:*****Numero de Error: 12Linea de codigo de error: =SaveSQL(WSFE.XmlResponse,"WSFE_CompConsultar"+cTipoCbte+"_"+cPunto_Vta+"_"+cCbte_Nro)Procedimiento: lote_procesoError VFP: No se encuentra la variable 'CTIPOCBTE'.Exepción: Traceback: 
28-04-2022 18:52:28 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,160)Fecha Ulti: 28-04-2022ERROR AL SOLICITARCAEEl nro de comprobante autorizado 160 ya se encontraba informado.El ultimo autorizado por Afip es 160El Total consultado (0.00)no coincide con el del comprobante (1839.20)
28-04-2022 18:55:16 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,160)Fecha Ulti: 28-04-2022WSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,160)Fecha Ulti: 28-04-2022602: No existen datos en nuestros registros para los parametros ingresados.
29-04-2022 11:09:02 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionSolicitando TATA Almacenado wsfeWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKLlamada: WSFE.CompConsultar602: No existen datos en nuestros registros para los parametros ingresados.
29-04-2022 11:12:58 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(0,0)11000: Campo PtoVta, no es un punto de venta valido, los valores posibles son entre 1 y 99998
11001: Campo CbteTipo, no es un tipo de comprobante valido. Ver metodo FEParamGetTiposCbte
29-04-2022 11:14:25 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(0,0)11000: Campo PtoVta, no es un punto de venta valido, los valores posibles son entre 1 y 99998
11001: Campo CbteTipo, no es un tipo de comprobante valido. Ver metodo FEParamGetTiposCbte
29-04-2022 11:16:35 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(0,0)11000: Campo PtoVta, no es un punto de venta valido, los valores posibles son entre 1 y 99998
11001: Campo CbteTipo, no es un tipo de comprobante valido. Ver metodo FEParamGetTiposCbte
29-04-2022 11:20:27 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(0,0)11000: Campo PtoVta, no es un punto de venta valido, los valores posibles son entre 1 y 99998
11001: Campo CbteTipo, no es un tipo de comprobante valido. Ver metodo FEParamGetTiposCbte
29-04-2022 11:24:01 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,1)Fecha Ulti: 28-04-2022Llamada: WSFE.CAESolicitarOrden:11000000076310058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
29-04-2022 11:25:55 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,1)Fecha Ulti: 28-04-2022Llamada: WSFE.CAESolicitarOrden:11000000076310058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
29-04-2022 11:27:11 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(8,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(8,3,1)Fecha Ulti: 28-04-2022Llamada: WSFE.CAESolicitarOrden:11000000076310058: De enviarse el tag CbtesAsoc debe enviarse PtoVta mayor a 0
29-04-2022 16:17:28 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(1,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(1,3,397)Fecha Ulti: 28-04-2022Llamada: WSFE.CAESolicitarCodigo error afip:*****Numero de Error: 1429Linea de codigo de error: cCAESolicitar = WSFE.CAESolicitar()Procedimiento: solicitar_procesoError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsfev1.pyc", line 288, in CAESolicitar
  File "pysimplesoap\client.pyc", line 148, in <lambda>
  File "pysimplesoap\client.pyc", line 352, in wsdl_call
  File "pysimplesoap\client.pyc", line 223, in call
SoapFault: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1208). ---> Input string was not in a correct format.
.Exepción: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1208). ---> Input string was not in a correct format.Traceback: Llamada: WSFE.CAESolictar Server was unable to read request. ---> There is an error in XML document (5, 1208). ---> Input string was not in a correct format.
29-04-2022 16:24:29 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OK
20-05-2022 19:44:07 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionSolicitando TATA Almacenado wsfeWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(3,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(3,3,32)Fecha Ulti: 20-05-2022Llamada: WSFE.CAESolicitarOrden:11000000741210040: Campo CbtesAsoc con tipo invalido, Tipo Valido  01, 02, 03, 04, 05, 34, 39, 60, 63, 88, 991.
14-09-2022 16:40:36 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsCertificados ProduccionSolicitando TATA Almacenado wsfeWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(2,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(2,3,0)Fecha Ulti: 14-09-2022Codigo error afip:*****Numero de Error: 1429Linea de codigo de error: cCAESolicitar = WSFE.CAESolicitar()Procedimiento: lote_procesoError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsfev1.pyc", line 288, in CAESolicitar
  File "pysimplesoap\client.pyc", line 148, in <lambda>
  File "pysimplesoap\client.pyc", line 352, in wsdl_call
  File "pysimplesoap\client.pyc", line 223, in call
SoapFault: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.
.Exepción: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.Traceback: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.
14-09-2022 16:46:27 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(2,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(2,3,0)Fecha Ulti: 14-09-2022Codigo error afip:*****Numero de Error: 1429Linea de codigo de error: cCAESolicitar = WSFE.CAESolicitar()Procedimiento: lote_procesoError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsfev1.pyc", line 288, in CAESolicitar
  File "pysimplesoap\client.pyc", line 148, in <lambda>
  File "pysimplesoap\client.pyc", line 352, in wsdl_call
  File "pysimplesoap\client.pyc", line 223, in call
SoapFault: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.
.Exepción: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.Traceback: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.
14-09-2022 16:55:48 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(2,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(2,3,0)Fecha Ulti: 14-09-2022Codigo error afip:*****Numero de Error: 1429Linea de codigo de error: cCAESolicitar = WSFE.CAESolicitar()Procedimiento: lote_procesoError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsfev1.pyc", line 288, in CAESolicitar
  File "pysimplesoap\client.pyc", line 148, in <lambda>
  File "pysimplesoap\client.pyc", line 352, in wsdl_call
  File "pysimplesoap\client.pyc", line 223, in call
SoapFault: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.
.Exepción: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.Traceback: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.
14-09-2022 16:56:52 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(2,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(2,3,0)Fecha Ulti: 14-09-2022Codigo error afip:*****Numero de Error: 1429Linea de codigo de error: cCAESolicitar = WSFE.CAESolicitar()Procedimiento: lote_procesoError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsfev1.pyc", line 288, in CAESolicitar
  File "pysimplesoap\client.pyc", line 148, in <lambda>
  File "pysimplesoap\client.pyc", line 352, in wsdl_call
  File "pysimplesoap\client.pyc", line 223, in call
SoapFault: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.
.Exepción: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.Traceback: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.
14-09-2022 17:29:52 , Iniciando FeedBackWSSA Version 2.11c WSSA InstallDir C:\APLICA~1\PyAfipWsTA recuperadoWSFE Version 1.23b WSFE InstallDir C:\APLICA~1\PyAfipWsEnviando TAConectando al servicio WSFEConexxion Existosaappserver status OKdbserver status OKauthserver status OKWSFE.CompUltimoAutorizado(2,3)Consultamos la fecha del ultimo comprobanteWSFE.CompConsultar(2,3,0)Fecha Ulti: 14-09-2022Codigo error afip:*****Numero de Error: 1429Linea de codigo de error: cCAESolicitar = WSFE.CAESolicitar()Procedimiento: lote_procesoError VFP: Código de excepción OLE IDispatch 0 de Python COM Server Internal Error: Unexpected Python Error: Traceback (most recent call last):
  File "win32com\server\policy.pyc", line 277, in _Invoke_
  File "win32com\server\policy.pyc", line 282, in _invoke_
  File "win32com\server\policy.pyc", line 585, in _invokeex_
  File "utils.pyc", line 157, in capturar_errores_wrapper
  File "wsfev1.pyc", line 288, in CAESolicitar
  File "pysimplesoap\client.pyc", line 148, in <lambda>
  File "pysimplesoap\client.pyc", line 352, in wsdl_call
  File "pysimplesoap\client.pyc", line 223, in call
SoapFault: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.
.Exepción: soap:Client: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.Traceback: Server was unable to read request. ---> There is an error in XML document (5, 1541). ---> Input string was not in a correct format.
